stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
That was sort of my original point - while I may be in the minority, I don't find the original to be wrong, let alone crude; in fact, FWIW, the change reads very oddly to me. Before anyone posts myriad links to style guides, a reminder that a manual of style represents decisions on grammar, not rules!

So for me personally, the path of least resistance would be no change here - but I don't strongly mind either way!